hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/MerkleTestBase.java

```diff
@@ -300,31 +300,28 @@ protected <T extends MerkleNode> T parseTree(@NonNull final byte[] state, @NonNu
 
     /** An implementation of {@link Codec} for String types */
     private static final class StringCodec implements Codec<String> {
+
         @NonNull
         @Override
-        public String parse(@NonNull DataInput input) throws IOException {
+        public String parse(@NonNull ReadableSequentialData input) {
             final var len = input.readInt();
             final var bytes = new byte[len];
             input.readBytes(bytes);
             return len == 0 ? "" : new String(bytes, StandardCharsets.UTF_8);
         }
 
+        @NonNull
         @Override
-        public void write(@NonNull String value, @NonNull DataOutput output) throws IOException {
-            final var bytes = value.getBytes(StandardCharsets.UTF_8);
-            output.writeInt(bytes.length);
-            output.writeBytes(bytes);
+        public String parseStrict(@NonNull ReadableSequentialData input) {
+            return parse(input);
         }
 
         @Override
-        public int measure(@NonNull DataInput input) throws IOException {
-            return input.readInt();
-        }
+        public void write(@NonNull String s, @NonNull WritableSequentialData output) {}
```

Comment:
Is this intentional to leave the method body blank ? Seems its overridden twice.  Did the IDE not show error ?